src/serde/ser.rs

```diff
@@ -60,19 +60,24 @@ pub fn node_to_stream<W: io::Write>(
             }
         }
     }
+
     Ok(())
 }
 
-pub fn node_to_bytes_limit(a: &Allocator, node: NodePtr, limit: usize) -> io::Result<Vec<u8>> {
+pub fn node_to_bytes_limit(
+    allocator: &Allocator,
```

Comment:
we actually have a convention throughout this code base, and `chia_rs` that we call the `Allocator` reference just `a`. I think that's a good convention because you end up needing to pass it to a lot of calls, and it quickly becomes a distraction to have a long name for it

Reply:
Fair enough, it is used quite often. Are you fine with the other names like `reader`/`writer`/`cursor` instead of `f` for clarity?